packages/devtools_app/lib/src/performance/timeline_analysis.dart

```diff
@@ -32,7 +33,8 @@ class TimelineAnalysisHeader extends StatelessWidget {
     final showFrameAnalysisButton = frameAnalysisSupported &&
         (controller.selectedFrame.value
                 ?.isUiJanky(controller.displayRefreshRate.value) ??
-            false);
+            false) &&
+        (!(controller.selectedFrame.value?.timelineEventData?.isEmpty ?? true));
```

Comment:
Nit: I think this would be clearer as `controller.selectedFrame.value?.timelineEventData?.isNotEmpty ?? false`

Reply:
good idea. done.